circle.yml

```diff
@@ -5,8 +5,27 @@ test:
   pre:
     - NODE_ENV=test make client/config/index.js:
         parallel: true
-    - case $CIRCLE_NODE_INDEX in 0) NODE_ENV=test make build-server ;; esac:
-        parallel: true
+    - ? |
+          case $CIRCLE_NODE_INDEX in
+            0) NODE_ENV=test make build-server;
+               ;;
+
+            1) I18N_DIR=$CIRCLE_ARTIFACTS/translate;
+               mkdir -p $I18N_DIR;
+               make translate;
+               mv calypso-strings.pot $I18N_DIR/calypso-strings.pot;
+               if [[ $CIRCLE_BRANCH != "master" ]]; then
+                 git merge-base --fork-point master | xargs git checkout;
+                 make translate;
+                 mv calypso-strings.pot $I18N_DIR/calypso-strings-master.pot;
+                 cp $I18N_DIR/calypso-strings-master.pot $I18N_DIR/calypso-strings-master-copy.pot;
+                 msgcat -u $I18N_DIR/calypso-strings*.pot > $I18N_DIR/localci-new-strings.pot;
+                 rm $I18N_DIR/calypso-strings-master-copy.pot;
+                 git checkout $CIRCLE_SHA1;
```

Comment:
This is just to bring everything back into the state it was before the `git checkout` above?